bugsnag-plugin-android-ndk/src/test/cpp/test_breadcrumbs.c

```diff
@@ -2,14 +2,12 @@
 #include <event.h>
 #include <time.h>
 
-bugsnag_breadcrumb *init_breadcrumb(const char *name, const char *message, bugsnag_breadcrumb_type type) {
+bugsnag_breadcrumb *init_breadcrumb(const char *name, char *message, bugsnag_breadcrumb_type type) {
```

Comment:
Can you expand to cover the new breadcrumb types?

Reply:
The serialization of new breadcrumb metadata types has been covered by an expansion of the `BreadcrumbStateSerializationTest` - I've retained this test as it already existed but it effectively duplicates coverage.